v2f/src/main/scala/org/broadinstitute/monster/etl/v2f/V2FConstants.scala

```diff
@@ -0,0 +1,244 @@
+package org.broadinstitute.monster.etl.v2f
+
+/**
+  * constants for v2f that allow for transforms of the converted tsv to json
+  */
+sealed trait V2FConstants {
+
+  /**
+    * the name of the tsv table that is being converted to Json and transformed
+    */
+  def tableName: String
+
+  /**
+    * relative path to the tsv that will be converted to Json and transformed
+    */
+  def tsvPath: String
+
+  /**
+    * the names of json fields that should be converted from a json string to a json double
+    */
+  def fieldsToConvertToJsonDouble: List[String]
+
+  /**
+    * the names of json fields that should be converted from a json string to a json double
```

Comment:
```suggestion
    * The names of fields that should be converted from a string to an integer.
```
Would also add a note on the 2nd line explaining why we need this one in addition to the "to double" method above.